src/parser/view_transform.c

```diff
@@ -5687,22 +5687,49 @@ mq_translate_local (PARSER_CONTEXT * parser, PT_NODE * statement, void *void_arg
     }
 
   /* resolve using index */
-  indexp = using_index;
+  indexp = using_index ? *using_index : NULL;
   if (indexp != NULL && spec != NULL)
     {
-      for (; indexp; indexp = indexp->next)
+      bool is_ignore = false;
+      PT_NODE *prev = NULL;
+      while (indexp)
 	{
-	  if (pt_resolve_using_index (parser, indexp, spec) == NULL)
+	  if (pt_resolve_using_index (parser, indexp, spec, &is_ignore))
```

Comment:
I would like to know the purpose of using is_ignore.
And I would like this description added as a comment.

Is_ignore in "else if" is for comparing the value set after the pt_resolve_using_index call?
It can be efficient because it does not compare when pt_resolve_using_index does not return null.
However, the flow is not easily understood.